### PR TITLE
Prepare for geo 0.13.0 and geo-types 0.5.0 release

### PR DESCRIPTION
**Pull request by urschrei**

### CODE REVIEW

File: CHANGES.md [hunk 1]
```diff
@@ -21,7 +21,7 @@
 * Remove usages of 'failure' crate
   * <https://github.com/georust/geo/pull/388>
 
-## geo-types (unreleased)
+## geo-types 0.4.4
```
Comment (frewsxcv) [inline CHANGES.md]: since there's a breaking change in here, we should bump to 0.5 here, and 0.13.0 in `geo`

Comment (urschrei) [reply CHANGES.md]: Oops. Done!

**Comment by urschrei** [comment]
@frewsxcv Can you push the release to crates when the CI runs complete? I know there's a small amount of manual intervention required in `geo`s `Cargo.toml` and I haven't done it before 😬

**Comment by frewsxcv** [comment]
Yep yep, I can take care of it

**Comment by urschrei** [comment]
bors retry

**Comment by frewsxcv** [comment]
published!

**Comment by urschrei** [comment]
ty!

**Pull request merged**